website/docs/sys-mgmt/data-exports.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Data Exports
+sidebar_label: Data Exports
+authentik_enterprise: true
+---
+
+The enterprise version of authentik allows you to export data (currently, users and events) in CSV format for backup or analysis purposes.
```

Comment:
can we clarify what type of data exactly, so to be clear that it is not PIP? Maybe something like:
"...allows you to export data (currently, non-PIP data about user and event objects) in CSV format..."?